agent/src/com/cloud/agent/Agent.java

```diff
@@ -685,31 +682,6 @@ protected void processRequest(final Request request, final Link link) {
         }
     }
 
-    private Answer setupDirectDownloadCertificate(SetupDirectDownloadCertificate cmd) {
```

Comment:
So, this is not used anymore?

Reply:
Yes, but moved to specific KVM code. Let me update the description